planex/cmd/mock.py

```diff
@@ -86,13 +89,20 @@ def createrepo(pkg_dir, metadata_dir, quiet=False):
     subprocess.check_call(cmd)
 
 
-def insert_loopback_repo(config_in_path, config_out_path, repo_path):
+def insert_loopback_repo(
+        config_in_path,
+        config_out_path,
+        repo_path,
+        extra_conf_list):
     """
     Write a new mock config, including a loopback repository configuration
     pointing to repo_path.    Ensure that the new config file's last-modified
     time is the same as the input file's, so that the mock chroot is not
     rebuilt.
     """
+    if extra_conf_list is None:
+        extra_conf_list = []
+
```

Comment:
if you define the default in the argument parser to be [] then you don't need to make this check